test/e2e/sources/googlecloudauditlogs/main.go

```diff
@@ -111,8 +111,9 @@ var _ = Describe("Google Cloud Audit Logs source", func() {
 				// FIXME: We observed that audit logs generated shortly after the source reports "Ready"
 				// weren't sent to the Pub/Sub topic observed by the receive adapter. It is likely that
 				// the Audit Logs Router Sink reconciled by the source starts routing audit logs after a
-				// delay (10-30s), so we virtually delay the next test steps here as well.
-				time.Sleep(30 * time.Second)
+				// delay (150s), so we virtually delay the next test steps here as well.
```

Comment:
From my experience, 10 sec is even enough sometimes. I had a few successful runs when I set this to 10s (but also a few failures).

I would keep the comment as `(10-150s)` to emphasize that the duration can vary a lot, and that we are planning for the worst case, not the "norm".

Reply:
That's right!